pilot/pkg/proxy/envoy/v2/monitoring.go

```diff
@@ -40,7 +40,13 @@ var (
 
 	edsInstances = monitoring.NewGauge(
 		"pilot_xds_eds_instances",
-		"Instances for each cluster, as of last push. Zero instances is an error.",
+		"Instances for each cluster(grouped by locality), as of last push. Zero instances is an error.",
+		monitoring.WithLabels(clusterTag),
+	)
+
+	edsEndpoints = monitoring.NewGauge(
+		"pilot_xds_eds_endpoints",
+		"Network endpoints for each cluster(across all localities), as of last push. Zero endpoints is an error.",
```

Comment:
oh I wish this was named differently. Or the previous one was named differently :(

Reply:
may be call this xds_eds_all_locality_endpoints?

Reply:
Yeah. I also felt the same confusion, earlier one would have been named differently. Since that is already in use, I will change it xds_eds_all_locality_endpoints - that makes sense.